pkg/internal/apis/certmanager/types_certificate.go

```diff
@@ -135,6 +140,21 @@ type CertificateSpec struct {
 	KeyEncoding KeyEncoding `json:"keyEncoding,omitempty"`
 }
 
+type X509DistinguishedName struct {
+	// Country/Region
+	Country            []string `json:"country,omitempty"`
+	Organization       []string `json:"organization,omitempty"`
+	OrganizationalUnit []string `json:"organizationalUnit,omitempty"`
+	// City
+	Locality []string `json:"locality,omitempty"`
+	// State/Province
+	Province      []string `json:"province,omitempty"`
+	StreetAddress []string `json:"streetAddress,omitempty"`
+	PostalCode    []string `json:"postalCode,omitempty"`
+	SerialNumber  string   `json:"serialNumber,omitempty"`
+	CommonName    string   `json:"commonName,omitempty"`
+}
```

Comment:
Same comments as other types - just a reminder to copy across the new changes once done 😄